### DISCUSSION TITLE
Issue #1150

### DISCUSSION THREAD

**Discussion by IvoLeal72** [cc-tweaked/CC-Tweaked#1173] [Q&A] [answered]
Hello, I would like to get started fixing #1150, but I am very new to cc code base, can someone point me where to look, so I can try to fix it?

**Comment by SquidDev** [comment, marked as answer]
Hi Ivo! Thanks for taking a look :).

File auto-complete is handled by the `fs.complete` function, which is defined in bios.lua:

https://github.com/cc-tweaked/CC-Tweaked/blob/da5956e943e4af9af168ac1c917bca1b84621002/src/main/resources/data/computercraft/lua/bios.lua#L716
